v2/pkg/engine/datasource/introspection_datasource/source.go

```diff
@@ -16,7 +17,7 @@ type Source struct {
 	introspectionData *introspection.Data
 }
 
-func (s *Source) Load(ctx context.Context, input []byte, w io.Writer) (err error) {
+func (s *Source) Load(ctx context.Context, input []byte, files []httpclient.File, w io.Writer) (err error) {
```

Comment:
Should we introduce "LoadWithFiles" which will only implemented by the GraphQL DataSource?
This would make the change less breaking and most DS don't have and will never have to worry about files.

Reply:
I too think we should go that route. Too many data sources affected. Will make the change

Reply:
Done